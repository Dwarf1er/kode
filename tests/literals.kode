SHOW 1.
SHOW 1.1.
SHOW -1.
SHOW -1.1.
SHOW TRUE.
SHOW FALSE.
SHOW "HELLO".
SHOW 'WORLD'.
