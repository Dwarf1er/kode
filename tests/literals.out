1
1.1
-1
-1.1
True
False
HELLO
WORLD
